### ISSUE TITLE
Wrong import path of npm library

### ISSUE THREAD

**Issue by masmau** [melfore/mosaic#62]
To import mosaic from an external application, `dist` must be appended.

Example:

```js
import { Button } from '@melfore/mosaic';
```
doesn't work.

```js
import { Button } from '@melfore/mosaic/dist';
```
works

**Comment by luciobordonaro** [comment]
Also this sounds a bit strange since I've used with the correct import previous versions of `mosaic` in kyp-web recipe branches. Maybe it's somehow related to #61

**Comment by luciobordonaro** [comment]
Found the issue and hopefully fixed in the PR targeting issue #61 .
See below for full context.

**Error:**
```
TypeScript error in /Users/lucio/Works/cra-test/src/App.tsx(3,24):
Could not find a declaration file for module '@melfore/mosaic'. '/Users/lucio/Works/cra-test/node_modules/@melfore/mosaic/dist/index.js' implicitly has an 'any' type.
  Try `npm install @types/melfore__mosaic` if it exists or add a new declaration (.d.ts) file containing `declare module '@melfore/mosaic';`  TS7016
```

**Reason:**
I didn't update the reference for declarations file to the correct path since the project setup.
Now declarations are exported in `/dist/index.d.ts` so I had to update this entry `"types": "./dist/index.d.ts"` in the `package.json` file

It was working on `kyp-recipe` imports in the feature branch because those were JS files, not requiring type declaration but only using the main entry of the library which is correctly set to `"main": "./dist/index.js"`.
